src/features/voters/components/VotersList.tsx

```diff
@@ -18,8 +18,9 @@ export const VotersList = (): JSX.Element => {
         Array(5)
           .fill(0)
           .map((_, i) => ({ recipient: i }))
-      ).map((voter) => (
-        <div key={voter.recipient}>
+      ).map((voter, i) => (
+        // eslint-disable-next-line react/no-array-index-key
```

Comment:
Adding index to key impact rendering performance

Reply:
there is a problem here: the `voter.recipient` could be the same, so the console would complain about `key duplicated`. that's why I need to add index as `i` to make the `key` looks different if the `voter.recipient` are the same. But, if I add index to key, it does impact rendering performance, not sure how to deal with this part.

fyi, the `voter` might only has `recipient`.

Reply:
should we prevent having > 1 of the same voter.recipient then?

Reply:
This list should contain only unique voters

Reply:
yes we should... so we don't need to deal with the duplicate problem now? since it might not be happened and need to be prevented else where?

Reply:
You can remove duplicated in voter list